Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -440,6 +441,39 @@ final class BuildPlanTests: XCTestCase {
       #endif
     }
 
+    func testSwiftCAsmMixed() throws {
+        let fs = InMemoryFileSystem(emptyFiles:
+            "/Pkg/Sources/exe/main.swift",
+            "/Pkg/Sources/lib/lib.c",
+            "/Pkg/Sources/lib/lib.S",
+            "/Pkg/Sources/lib/include/lib.h"
+        )
+
+        let diagnostics = DiagnosticsEngine()
+        let graph = loadPackageGraph(root: "/Pkg", fs: fs, diagnostics: diagnostics,
+                                     manifests: [
+                                        Manifest.createV4Manifest(
```

Comment:
I added a `Manifest.createManifest` method for generic manifests

Reply:
awesome, thanks, changed